flutter/lib/src/default_integrations.dart

```diff
@@ -209,15 +209,16 @@ class NativeSdkIntegration extends Integration<SentryFlutterOptions> {
         'attachStacktrace': options.attachStacktrace,
         'attachThreads': options.attachThreads,
         'autoSessionTrackingIntervalMillis':
-            options.autoSessionTrackingIntervalMillis,
+            options.autoSessionTrackingInterval.abs().inMilliseconds,
```

Comment:
why do we need to `abs`?

Reply:
Since there are asserts now, this is not needed anymore